### ISSUE TITLE
Entries' Bulk Action dropdown remains disabled unless page is refreshed.

### ISSUE THREAD

**Issue by bucketpress** [CalderaWP/Caldera-Forms#1833]
This occurs when managing entries of a form:
- At 'Active' tab, I select multiple entries, select a bulk action to move to trash and apply.
- Bulk Action dropdown becomes disabled
- I then go to the 'Trash' tab, I select multiple entiries, but Bulk Action dropdown remains disabled, and I can't apply anything.

Seems like once Bulk Action is applied once, it remains disabled no matter which form or tab I go to.

Using the following versions:
WordPress Version: 4.8.1
PHP Version: 5.6.31
Caldera Forms: 1.5.4

**Comment by blunket** [comment]
Can confirm that I've had the same issue

**Comment by blunket** [comment]
pretty simple fix! all it took was moving the code that re-enables the dropdown to the top of the callback function. @bucketpress see my PR if you'd like

**Comment by blunket** [comment]
@AlchemyUnited  Nope, it was not merged. Actually though I cannot even find my PR in this repo at all.

**Comment by AlchemyUnited** [comment]
@blunket  - Whoa. That's not good. I'm gotta open a new ticket (to get this to the top of the queue).  You still got the PR sitting around? :) Maybe you can re-submit?

**Comment by AppWebIT** [comment]
This fix is simple, but the Bulk Action dropdown is then always enabled - i've fixed it for myself in 

`function setup_pagination(obj)`

after // setup values with

`jQuery('#cf_bulk_action').removeAttr('disabled');` 

Whenever the list is reloaded, the dropdown will be reactivated.

**Comment by AppWebIT** [comment]
Of course you can also use this expression because the variable is declared above

`bulk_actions.removeAttr('disabled');`

**Comment by blunket** [comment]
@AppWebIT did you see my PR? not sure why they never merged it, I don't know if it's even accurate anymore?

**Comment by AppWebIT** [comment]
@blunket I've seen your PR - before I looked for the problem on Github, I tried to find a solution for myself. At first I resolved it your way, but the dropdown is immediately active. That's why I added the code somewhere else. So the dropdown will be activated after loading. Since I'm not really active here on GitHub, unfortunately I do not know how to report a problem. I just wanted to share my experiences. It's strange that the problem has existed for so long and nobody does anything about it.
